src/libcore/num/mod.rs

```diff
@@ -1601,6 +1601,29 @@ $EndFeature, "
             }
         }
 
+        doc_comment! {
+            concat!("Wrapping (modular) absolute value. Computes `self.abs()`, without any wrapping
+or panicking.
+
+
+# Examples
+
+Basic usage:
+
+```
+", $Feature, "#![feature(unsigned_abs)]
+assert_eq!(100", stringify!($SelfT), ".unsigned_abs(), 100", stringify!($UnsignedT), ");
+assert_eq!((-100", stringify!($SelfT), ").unsigned_abs(), 100", stringify!($UnsignedT), ");
+assert_eq!((-128i8).unsigned_abs(), 128u8);",
```

Comment:
Why does this line not use `$SelfT`? Won't the documentation for non-`i8` be confusing? Can you use `$Min` / `$Max` instead?

Reply:
I copied that case from `wrapping_abs`. There's no way to get $MIN and -$MIN (as unsigned) and have that appear as an actual number.